src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx

```diff
@@ -1,6 +1,7 @@
 import {pick} from 'lodash';
 
 import {DEFAULT_PER_PAGE} from 'app/constants';
+import {URL_PARAM} from 'app/components/organizations/globalSelectionHeader/constants';
```

Comment:
should this be moved out of this component specific dir?

Reply:
Moved!